README.md

```diff
@@ -31,6 +31,23 @@ This command will:
 7. dump all the keys and values into the `.env` file
 
 
+## Advanced Usage
+```bash
+$ dump-env -t .env.template -p 'SECRET_ENV_' -p 'ANOTHER_SECRET_ENV_' > .env
+```
+
+This command will do the same as in quickstart demo but variables with second prefix will replace variables with first prefix.
```

Comment:
Are you sure about this wording?

Reply:
Maybe it is better to rephrase it to:
```
## Multiple prefixes

.. example

This command will do pretty much the same thing as with one prefix. But, it will replace multiple prefixes.
```

```bash
$ export SECRET_TOKEN='very secret string'
$ export ANOTHER_SECRET_ENV_VALUE='0'
$ export SECRET_ANSWER='13'
$ export ANOTHER_SECRET_ENV_ANSWER='42'
$ dump-env -p SECRET_ -p ANOTHER_SECRET_ENV_
ANSWER=42
TOKEN=very secret string
VALUE=0
```

What order should user expect? Who would win? `SECRET_` value or `ANOTHER_SECRET_ENV_` value?

Reply:
Makes sense. I'll fix that.

Reply:
@sobolevn how about now?